src/views/preview/presentation.jsx

```diff
@@ -94,7 +94,6 @@ const PreviewPresentation = ({
                                 </a>
                                 <div className="title">
                                     {editable ?
-
```

Comment:
Not sure why this line is missing here, must have happend when solving conflicts. Should I add the line back?

Reply:
This is fine.